docs/docs/integrations/llms/azure_ml.ipynb

```diff
@@ -45,7 +45,7 @@
     "* `GPT2ContentFormatter`: Formats request and response data for GPT2\n",
     "* `DollyContentFormatter`: Formats request and response data for the Dolly-v2\n",
     "* `HFContentFormatter`: Formats request and response data for text-generation Hugging Face models\n",
-    "* `LLamaContentFormatter`: Formats request and response data for LLaMa2\n",
+    "* `OpenAIStyleContentFormatter`: Formats request and response data for open source models like LLaMa2 that follow OpenAI API scheme.\n",
```

Comment:
They are not necessary open source models. So we should say "OpenAI compatible APIs"

Reply:
Fixed